packages/common/src/components/TableView/sort.ts

```diff
@@ -86,26 +88,30 @@ export const useSort = (
  */
 export const buildSort = ({
   columnIndex,
-  columns,
+  resourceFields,
   activeSort,
   setActiveSort,
 }: {
   columnIndex: number;
-  columns: Column[];
+  resourceFields: ResourceField[];
   activeSort: SortType;
   setActiveSort: (sort: SortType) => void;
 }): ThSortType => ({
   sortBy: {
     index:
-      columns.find(({ id }) => id === activeSort.id) &&
-      columns.findIndex(({ id }) => id === activeSort.id),
+      resourceFields.find(
```

Comment:
well for a new comer to this area of the code, I found a lot of `id` and `field`  that refer to different components ids and fields and labels, it does not add value to a person that know the code, but it help help me navigate the code.

Reply:
IMHO in case of generic components adding domain names only adds extra noise.
In the case above `resource` prefix will rather slow down the new comer.